packages/waku/src/lib/builder/build.ts

```diff
@@ -120,6 +124,7 @@ const analyzeEntries = async (rootDir: string, config: ResolvedConfig) => {
       noExternal: /^(?!node:)/,
     },
     build: {
+      emptyOutDir: !partial,
```

Comment:
Do we need this with `write: false`?

Reply:
This was there before I joined. I suppose because `analyzeEntries` does not have to create a bundle?

Reply:
Yes, I mean, as we have `write: false`, `emtpyOutDir: !partial` might be unnecessary. Can you try?